source/resultTypes.ts

```diff
@@ -162,4 +159,133 @@ export interface eventResult {
       year?: number
     }
   ]
+}
+
+export interface titleItem {
+  title: string,
+  page_id: number,
+  rev: number,
+  tid: number,
+  namespace: number,
+  user_id: number,
+  user_text: string,
+  timestamp: string,
+  comment: string,
+  tags: Array<string>,
+  restrictions: Array<string>,
+  page_language: string,
+  redirect: boolean
+}
+
+export interface title {
+  items: Array<titleItem>
+}
+
+export interface related {
+  pages: Array<relatedItem>
+}
+
+export interface relatedItem {
```

Comment:
I think we can use the wikiSummary interface here instead of the relatedItem. Both look almost the same and any fields you feel is missing here should be added to the wikiSummary interface.

Reply:
Fair point. I removed relatedItem and added the missing fields to wikiSummary.